src/theme/useTheme.js

```diff
@@ -1,22 +1,23 @@
-import { useMemo, useLayoutEffect } from "react";
+import { useMemo, useLayoutEffect, useState } from "react";
 import deepmerge from "deepmerge";
 import lightTheme from "./lightTheme";
 
-const useTheme = (themeOverrides) => {
+const useTheme = () => {
+  const [theme, setTheme] = useState(null);
```

Comment:
@fernandoabolafio 
For some reason, theme is reset to initial value `null` when calling `useTheme()` which causes the `useMemo` to rerun and calculate new value and default light theme is overriding theme values again, what am I missing :thinking: ?